load-generator-deployment.yml

```diff
@@ -44,11 +44,11 @@ spec:
           stdin: true
           resources:
             requests:
-              cpu: "0.5"
-              memory: "2Gi"
+              cpu: "3"
+              memory: "10Gi"
```

Comment:
What nodes are we using to run this on? They'll require high memory to fit these pods.